_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
```

Comment:
@ZackButcher Any recommendations for this bullet?

Reply:
**Incremental Istio Deployment (Preview)**: You can now adopt Istio incrementally, more easier than before, by installing only the components you want (e.g, Pilot+Ingress only as the minimal Istio install). Refer to `istioctl` CLI tool for generating a customized Istio deployment.